haiku-misc/neonlights/neonlights-0.0.1.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Screensaver drawing neon lines"
+DESCRIPTION="NeonLights is a screensaver that draws colorful neon lines. \
+It was based loosely on Jared Tarbell's work. Inspired by Atalax's Substrate."
+HOMEPAGE="https://github.com/pulkomandy/neonlights/"
+COPYRIGHT="2017 Adrien Destugues"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="f658b45894fe5d2349be4e06c3a0bce359985e27"
+SOURCE_URI="https://github.com/pulkomandy/neonlights/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9271f3778218bb63430da00d00b099c5d3e9d83b21d6c132533069b237199663"
+SOURCE_DIR="neonlights-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	neonlights = $portVersion
+	addon:NeonLights
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc
+	"
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	mkdir -p $addOnsDir/Screen\ Savers/
+	cp objects/"Neon Lights" $addOnsDir/Screen\ Savers/
```

Comment:
You might wish to replace
```
	mkdir -p $addOnsDir/Screen\ Savers/
	cp objects/"Neon Lights" $addOnsDir/Screen\ Savers/
```
by
```
	install -d $addOnsDir/Screen\ Savers
	install -t $addOnsDir/Screen\ Savers objects/"Neon Lights" 
```
This is not required, but since it looks nicer and is equivalent to the mkdir + cp, I would recommend that change.

Hmm, it might be even nicer to use
```
	install -d "$addOnsDir/Screen Savers"
	install -t "$addOnsDir/Screen Savers" "objects/Neon Lights"
```
or
```
	install -d $addOnsDir/Screen\ Savers
	install -t $addOnsDir/Screen\ Savers objects/Neon\ Lights
```
(You can chose whatever you wish, or even keep the current INSTALL.)